ddl/ddl.go

```diff
@@ -1249,12 +1248,6 @@ var (
 	RunInGoTest bool
 )
 
-func init() {
-	if flag.Lookup("test.v") != nil || flag.Lookup("check.v") != nil {
-		RunInGoTest = true
```

Comment:
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/3427324/213841991-f94b0d15-fadc-487e-9a71-1e11dc7a9948.png">

we have many of the same codes. I think we can unit it.

Reply:
I fixed that because of the init function. In addition, we set `ddl.RunInGoTest`.
For example, here you list `sanity_check.go line185` with no variables.